go-controller/pkg/node/management-port.go

```diff
@@ -58,6 +65,55 @@ func (n *OvnNode) createManagementPort(hostSubnets []*net.IPNet, nodeAnnotator k
 		return err
 	}
 
+	if config.Gateway.Mode == config.GatewayModeLocal {
+		var gatewayIfAddrs []*net.IPNet
+		for _, hostSubnet := range hostSubnets {
+			// local gateway mode uses mp0 as default path for all ingress traffic into OVN
+			var nextHop *net.IPNet
+			if utilnet.IsIPv6CIDR(hostSubnet) {
+				nextHop = cfg.ipv6.ifAddr
+			} else {
+				nextHop = cfg.ipv4.ifAddr
+			}
+			gatewayIfAddrs = append(gatewayIfAddrs, nextHop)
+			stdout, stderr, err = util.RunOVNNbctl("--may-exist",
+				"--policy=src-ip", "lr-route-add", ovnClusterRouter,
+				hostSubnet.String(), nextHop.IP.String())
+			if err != nil {
+				return fmt.Errorf("failed to add source IP address based "+
+					"routes in distributed router %s, stdout: %q, "+
+					"stderr: %q, error: %v", ovnClusterRouter, stdout, stderr, err)
+			}
+			// add iptables masquerading for mp0 to exit the host for egress
+			cidr := nextHop.IP.Mask(nextHop.Mask)
+			cidrNet := &net.IPNet{IP: cidr, Mask: nextHop.Mask}
+			err = initLocalGatewayNATRules(util.K8sMgmtIntfName, cidrNet)
+			if err != nil {
+				return fmt.Errorf("failed to add local NAT rules for: %s, err: %v", util.K8sMgmtIntfName, err)
+			}
+		}
+
+		n.initLocalEgressIP(gatewayIfAddrs, util.K8sMgmtIntfName)
+
+		ips, err := getNetworkInterfaceIPAddresses(util.K8sMgmtIntfName)
```

Comment:
util.GetNodeManagementIfAddr(subnet *net.IPNet)

Reply:
I actually don't think any of this is necessary anymore. This was leftover from the original hybrid PR.